src/sentry/search/utils.py

```diff
@@ -51,74 +51,126 @@ def parse_datetime_range(value):
         raise InvalidQuery(u'{} is not a valid datetime query'.format(value))
 
     if flag == '-':
-        return (timezone.now() - delta, None)
+        return (
+            (timezone.now() - delta, True),
+            None,
+        )
     else:
-        return (None, timezone.now() - delta)
+        return (
+            None,
+            (timezone.now() - delta, True),
+        )
 
 
-def parse_datetime_comparison(value):
-    # TODO(dcramer): currently inclusitivity is not controllable by the query
-    # as from date is always inclusive, and to date is always exclusive
-    if value[:2] in ('>=', '=>'):
-        return (parse_datetime_value(value[2:])[0], None)
-    if value[:2] in ('<=', '=<'):
-        return (None, parse_datetime_value(value[2:])[0])
```

Comment:
The messed up operators here (`>=`, `=<`) couldn't actually be hit because they weren't specified in `parse_datetime_expression` so I removed them.